.gitlab/deps_fetch/deps_fetch.yml

```
@@ -57,7 +57,7 @@ go_deps:
         files:
           - \**/go.mod
           - .gitlab/deps_fetch/deps_fetch.yml
-        prefix: "go_deps"
+        prefix: "go_deps_v2"
```

```suggestion
        prefix: "go_deps_modcache"
```
🥜 we can use a meaningful prefix instead of `v2`